<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>base_K3bSetup</class>
 <widget class="QWidget" name="base_K3bSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QHBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QGroupBox" name="groupBox3">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QCheckBox" name="m_checkUseBurningGroup">
            <property name="whatsThis">
             <string>&lt;p&gt;If this option is checked, only the users in the specified group will be able to burn CDs and DVDs, since only they will have access to the devices and the CD recording programs used by K3b.&lt;/p&gt;
&lt;p&gt;Otherwise all users on the system have access to the devices and to all K3b functionality.</string>
            </property>
            <property name="text">
             <string>Use burning group:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KLineEdit" name="m_editBurningGroup">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>burning</string>
            </property>
            <property name="showClearButton" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="m_frameUsers">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <spacer name="spacer1">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="textLabel2">
             <property name="text">
              <string>Users allowed to burn (separated by space):</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KLineEdit" name="m_editUsers">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="showClearButton" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox1">
       <property name="title">
        <string>Devices</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="text">
           <string>Check the devices whose permissions you want to be changed</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="m_viewDevices">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <attribute name="headerStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox2">
       <property name="title">
        <string>External Programs</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Found Programs</string>
           </attribute>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="textLabel1">
              <property name="text">
               <string>Check the programs whose permissions you want to be changed</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeView" name="m_viewPrograms">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerStretchLastSection">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Search Path</string>
           </attribute>
           <layout class="QVBoxLayout">
            <item>
             <widget class="KEditListBox" name="m_editSearchPath">
              <property name="title">
               <string>Search Path</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel1_3">
              <property name="text">
               <string>&lt;qt&gt;&lt;b&gt;Hint:&lt;/b&gt; to force K3b to use another than the default name for the executable specify it in the search path.&lt;/qt&gt;</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">keditlistbox.h</include>
  <include location="local">klineedit.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_checkUseBurningGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_editBurningGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkUseBurningGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_editUsers</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
